### PR TITLE
{bio}[goolf/1.7.20] Hisat2 (Review)

### PR DESCRIPTION
**Pull request by npklein**

### CODE REVIEW

**Comment by npklein** [comment]
Test report by @npklein
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz, Python 2.7.9
See https://gist.github.com/bd97ca4827766a84d0fa for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+easyblock = 'MakeCp'
+
+name = 'hisat2'
+version = '2.0.0-beta'
+
+homepage = 'http://ccb.jhu.edu/software/hisat/index.shtml'
```
Comment (boegel) [inline easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb]: should be http://ccb.jhu.edu/software/hisat2/index.shtml ?

File: easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+description = """HISAT is a fast and sensitive spliced alignment program for mapping RNA-seq reads. In addition to one
```
Comment (boegel) [inline easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb]: also needs an update?

File: easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+easyblock = 'MakeCp'
+
+name = 'hisat2'
```
Comment (boegel) [inline easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb]: should be `HISAT2`?

File: easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+homepage = 'http://ccb.jhu.edu/software/hisat/index.shtml'
+description = """HISAT is a fast and sensitive spliced alignment program for mapping RNA-seq reads. In addition to one 
+ global FM index that represents a whole genome, HISAT uses a large set of small FM indexes that collectively cover 
+ the whole genome (each index represents a genomic region of ~64,000 bp and ~48,000 indexes are needed to cover the 
+ human genome). These small indexes (called local indexes) combined with several alignment strategies enable effective 
+ alignment of RNA-seq reads, in particular, reads spanning multiple exons. The memory footprint of HISAT is relatively 
+ low (~4.3GB for the human genome). We have developed HISAT based on the Bowtie2 implementation to handle most of the 
+ operations on the FM index. """
+
+sources = ['%(name)s-%(version)s-source.zip']
```
Comment (boegel) [inline easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb]: namelower?

File: easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
```
Comment (boegel) [inline easybuild/easyconfigs/h/hisat2/hisat2-2.0.0-beta-goolf-1.7.20.eb]: this should be in between `description` and `source_urls`/`sources`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5ce1a0a8b9be67f116fe for a full test report.

**Comment by migueldiascosta** [comment]
@npklein, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**